third_party/lang-items/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-libtock_core = { path = "../../third_party/libtock-rs/core", default-features = false, features = ["alloc_init", "custom_panic_handler", "custom_alloc_error_handler"] }
+libtock_core = { path = "../../third_party/libtock-rs/core", default-features = false, features = ["custom_panic_handler", "custom_alloc_error_handler"] }
```

Comment:
I guess that's the solution to the problem we faced.

Reply:
Yes :')